odl/contrib/datasets/ct/mayo.py

```diff
@@ -126,30 +123,39 @@ def load_projections(folder, indices=None):
     src_radius = datasets[0].DetectorFocalCenterRadialDistance
     det_radius = (datasets[0].ConstantRadialDistance -
                   datasets[0].DetectorFocalCenterRadialDistance)
-
-    # Convert pitch and offset to odl defintions
     pitch = (pixel_size[1] * shape[1] * datasets[0].SpiralPitchFactor *
              src_radius / (src_radius + det_radius))
-    offset_along_axis = (datasets[0].DetectorFocalCenterAxialPosition -
-                         angles[0] / (2 * np.pi) * pitch)
 
     # Get flying focal spot data
     offset_axial = np.array([d.SourceAxialPositionShift for d in datasets])
     offset_angular = np.array([d.SourceAngularPositionShift for d in datasets])
     offset_radial = np.array([d.SourceRadialDistanceShift for d in datasets])
 
-    angles_offset = angles - offset_angular
-    src_rad_offset = src_radius + offset_radial
-    offset_x = (np.cos(angles_offset) * (-src_rad_offset) -
-                np.cos(angles) * (-src_radius))
-    offset_y = (np.sin(angles_offset) * (-src_rad_offset) -
-                np.sin(angles) * (-src_radius))
-    offset_z = offset_axial
-
-    # TODO: WE CURRENTLY IGNORE THE OFFSETS DUE TO FLYING FOCAL SPOT
-    source_offsets = np.array([offset_x, offset_y, offset_z]).T
+    if 1:
+        # Apply only mean of offsets
+        src_radius = src_radius + np.mean(offset_radial)
```

Comment:
I don't get this. How could this ever work before if it's a vector? In the old version, `src_rad_offset` must have been a vector, and with it all 3 offsets as well.

Reply:
Basically the offsets are _weird_, so I decided to just go for their means. Before we did not do anything about it (src_rad_offset  was unused) which gave some slight errors, since apparently the "offsets" are not zero-centered.

Reply:
Okay. What I was puzzled about was that in the old version, `src_radius` became a vector (after adding `offset_radial`), then it was used to compute `offset_[xyz]`, each of which also was a vector then, and the final `source_offsets` stacked them to a matrix. I don't see how that made sense.

Anyhow, for some other PR.